roles/matrix-nginx-proxy/templates/nginx/conf.d/matrix-synapse.conf.j2

```diff
@@ -120,6 +120,32 @@ server {
 		proxy_max_temp_file_size 0;
 	}
 
+    {% if matrix_synapse_enable_metrics %}
```

Comment:
Should we continue with tabs? According to https://yaml.org/spec/1.2/spec.html in section 6.1 it is recommended to use spaces and not tabs to maintain portability.  

It would be nice to add a `.editorconfig` to the project as well.

Reply:
Using spaces in YAML is preferred, that's right.

This is not a YAML file though, but an nginx configuration file.

Indeed, an `.editorconfig` would be nice to have. It has already been mentioned once in #83.

Reply:
Yes, I know. and it's not a big deal, but I would prefer a common denominator that works for all files.

Reply:
I'm not aware of any configuration format that forces people to use "only tabs" (like YAML forces people to use only spaces), so this common denominator automatically becomes "spaces". Especially with the unfortunate prevalence that YAML has in the Ansible world and the various Python-based projects.

Some people would say "use spaces everywhere", others would say "use tabs everywhere" (well, unless it's YAML which forces you to use spaces). I'm in the camp that thinks it's better to not spread YAML's space-extortion more than is required, balancing it with a fair amount of tabs wherever possible, just because we have that freedom :laughing:

Reply:
That's totally fine with me despite I'm in the "use spaces everywhere"-camp. :smile:

Will something like this work as an `.editorconfig` then?

```
# This file is the top-most EditorConfig file
root = true

# All Files
[*]
charset = utf-8
end_of_line = lf
indent_style = tab
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true

#########################
# File Extension Settings
#########################

# YAML Files
[*.{yml,yaml}]
ident_style = spaces
indent_size = 2

# Markdown Files
[*.md]
trim_trailing_whitespace = false

# 4 space indentation
[*.{ini,py,py.tpl,rst}]
indent_style = space
indent_size = 4
```

Reply:
Absolutely, I'm creating a new PR.